quantum/process_keycode/process_unicodemap.c

```diff
@@ -49,7 +49,7 @@ bool process_unicode_map(uint16_t keycode, keyrecord_t *record) {
   if ((keycode & QK_UNICODE_MAP) == QK_UNICODE_MAP && record->event.pressed) {
     const uint32_t* map = unicode_map;
     uint16_t index = keycode - QK_UNICODE_MAP;
-    uint32_t code = pgm_read_dword_far(&map[index]);
+    uint32_t code = pgm_read_dword(&map[index]);
```

Comment:
My understanding of this is that `pgm_read_dword_far` will use the 32 bit address, whereas the `pgm_read_dword` will only use the 16 bit - is there any reason for this change?